### ISSUE TITLE
DL4J: MKLDNN batch norm backprop support

### ISSUE THREAD

**Issue by AlexDBlack** [eclipse/deeplearning4j#8172]
Currently DL4J supports MKL-DNN for batch norm forward pass only. Backprop needs to be added - requires this first: https://github.com/eclipse/deeplearning4j/issues/8171

**Comment by shyrma** [comment]
Hi, Alex
MKLDNN api support for batch norm backprop has been provided and merged

**Comment by AlexDBlack** [comment]
Blocked on: https://github.com/eclipse/deeplearning4j/issues/8335
